app/src/main/java/com/cornellappdev/android/eateryblue/ui/components/home/EateryHourBottomSheet.kt

```diff
@@ -0,0 +1,159 @@
+import com.cornellappdev.android.eateryblue.ui.theme.Green
+import com.cornellappdev.android.eateryblue.ui.theme.Red
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun EateryHourBottomSheet(
+    eatery: Eatery,
+    onDismiss: () -> Unit,
+    onReportIssue: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 14.dp)
+            .padding(16.dp),
+        horizontalAlignment = Alignment.Start
+    ) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Row {
+                Icon(
+                    imageVector = Icons.Outlined.Schedule,
+                    contentDescription = "Hours Icon",
+                    tint = Color.Black,
+                    modifier = Modifier.size(32.dp)
+                )
+                Spacer(modifier = Modifier.size(ButtonDefaults.IconSpacing))
+                androidx.compose.material.Text(
+                    text = "Hours",
+                    style = EateryBlueTypography.h4,
+                    color = Color.Black
+                )
+            }
+            IconButton(
+                onClick = {
+                    onDismiss()
+                },
+                modifier = Modifier
+                    .size(40.dp)
+                    .background(color = GrayZero, shape = CircleShape)
+            ) {
+                Icon(
+                    Icons.Default.Close,
+                    contentDescription = Icons.Default.Close.name,
+                    tint = Color.Black
+                )
+            }
+        }
+
+        Spacer(modifier = Modifier.height(12.dp))
+
+        val openUntil = eatery.getOpenUntil()
+        androidx.compose.material.Text(
+            modifier = Modifier.padding(top = 2.dp),
+            text = openUntil ?: "Closed",
+            style = TextStyle(
+                fontWeight = FontWeight.SemiBold, fontSize = 16.sp
+            ),
+            color = if (openUntil.isNullOrBlank()) Red else Green,
+        )
+
+        Spacer(modifier = Modifier.height(12.dp))
+
+        val operatingHours = eatery.formatOperatingHours()
+
+        operatingHours.forEachIndexed { index, (dayRange, hours) ->
+            Column {
+                Text(text = dayRange, fontSize = 16.sp, color = GrayFive)
+                hours.forEach { hour ->
+                    Text(
+                        text = hour,
+                        fontSize = 18.sp,
+                        color = Color.Black,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+                if (index != operatingHours.lastIndex) {
+                    Spacer(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .height(12.dp)
+                    )
+                }
```

Comment:
I like the extra caution to make spacing equal! But I actually think all of the rows should have a space; even the last entry seems to have 12 padding on the Figma:

![image](https://github.com/cuappdev/eatery-blue-android/assets/47724806/df839ee8-3524-4cff-9741-e5b665386961)

So you should be able to just turn this into a regular `.forEach`.